pkg/controller/master/backup/backup.go

```diff
@@ -546,7 +556,7 @@ func (h *Handler) deleteVMBackupMetadata(vmBackup *harvesterv1.VirtualMachineBac
 
 	destURL := filepath.Join(metadataFolderPath, getVMBackupMetadataFileName(vmBackup.Namespace, vmBackup.Name))
 	if exist := bsDriver.FileExists(destURL); exist {
-		logrus.Infof("delete vm backup metadata %s/%s inbackup target %s", vmBackup.Namespace, vmBackup.Name, target.Type)
+		logrus.Infof("delete vm backup metadata %s/%s in backup target %s", vmBackup.Namespace, vmBackup.Name, target.Type)
```

Comment:
probably moving those logs to `debug` mode is sufficient enough?

Reply:
Updated it. Thank you.